### DISCUSSION TITLE
Welcome to logitorch Discussions!

### DISCUSSION THREAD

**Discussion by ChadiHelwe** [ChadiHelwe/logitorch#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ke-lara** [comment]
Hi, This is Raja from Singapore. Our team works on Interpretable Logic Reasoning. We just published a paper on our Chainformer model that outperforms humans on LogicNLI. 

Investigating Transformer-Guided Chaining for Interpretable Natural Logic Reasoning
Kanagasabai Rajaraman, Saravanan Rajamanickam, Wei Shi
https://aclanthology.org/2023.findings-acl.588/

Can I request to add to your excellent list? We plan to release codes once we get internal clearance.

**Reply by ChadiHelwe** [reply]
Hello Raja, yes sure!

It will be great if you can implement the model into the library.
